apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -103,19 +92,16 @@ export function ColumnToggleButton() {
             <FormControl>
                 <Select
                     multiple
-                    value={activeColumns}
+                    value={selectedColumnNames}
                     open={open}
                     onChange={handleChange}
                     onClose={handleClose}
                     renderValue={renderEmptySelectValue}
                     sx={{ visibility: 'hidden' }}
                 >
-                    {Object.entries(columnLabels).map(([column, label]) => (
+                    {Object.entries(columnLabels).map(([column, label], idx) => (
```

Comment:
Can we compute the object entries for column labels at the top level, this saves on some computation for every render.

```suggestion
                    {columnLabelEntries.map(([column, label], index) => (
```